.github/workflows/build.yml

```
@@ -33,7 +33,7 @@ jobs:
       - metadata
       - npm-test
     if: inputs.build-type == 'release'
-    uses: howsoai/.github/.github/workflows/npm-release.yml@main
+    uses: howsoai/.github/.github/workflows/npm-release-open-source.yml@21221-npm-open-source
```

```suggestion
    uses: howsoai/.github/.github/workflows/npm-release-open-source.yml@main
```

Though, that needs merged first.

---

https://github.com/howsoai/.github/pull/9